client/src/components/flashcard/AddFlashcard.tsx

```diff
@@ -90,83 +103,148 @@ const AddFlashcard = (): ReactElement => {
         } else {
             setAnswerError(false);
         }
-        setAnswer(value);
+        setCorrectAnswer(value);
+    };
+
+    const handleIncorrectAnswer = () => {
+        if (!incorrectAnswer) return;
+        setAnswers([...answers, incorrectAnswer]);
+        setIncorrectAnswer('');
+    };
+
+    const handleDeleteAnswer = (index: number) => () => {
+        setAnswers(answers.filter((el, ind) => ind !== index));
+    };
+
+    const handleIncorrectAnswerChange = (answer: string, index: number) => {
+        setAnswers(
+            answers.map((el, ind) => {
+                if (ind !== index) return el;
+                return answer;
+            })
+        );
     };
 
     return (
         <>
-            <Container maxWidth="sm">
-                <Box m={4} textAlign="center">
-                    <Typography variant="h2">TWORZENIE FISZKI</Typography>
-                </Box>
-                <Box m={4}>
-                    <TextField
-                        error={promptError}
-                        fullWidth
-                        label="Pytanie"
-                        value={prompt}
-                        helperText="To pole jest wymagane"
-                        onChange={(e) => handleSetPrompt(e.target.value)}
-                    />
-                </Box>
-                <Box m={4}>
-                    <TextField
-                        error={answerError}
-                        fullWidth
-                        label="Odpowiedź"
-                        value={answer}
-                        helperText="To pole jest wymagane"
-                        onChange={(e) => handleSetAnswer(e.target.value)}
-                    />
-                </Box>
-                <Box m={4}>
-                    <TextField
-                        value={extraInfo}
-                        fullWidth
-                        label="Dodatkowe informacje"
-                        onChange={(e) => setExtraInfo(e.target.value)}
-                    />
-                </Box>
+            <Grid container direction="column" justify="center" alignItems="center" sm={12}>
```

Comment:
konsola się dowala, że zbędne
```suggestion
            <Grid container direction="column" justify="center" alignItems="center">
```